netebpfext/net_ebpf_ext.c

```diff
@@ -996,4 +1010,24 @@ net_ebpf_ext_remove_client_context(
     }
 
     ExReleaseSpinLockExclusive(&filter_context->lock, old_irql);
-}
\ No newline at end of file
+}
+
+#ifdef _DEBUG
+void
+net_ebpf_ext_add_filter_context_to_zombie_list(_In_ net_ebpf_extension_wfp_filter_context_t* filter_context)
```

Comment:
```suggestion
net_ebpf_ext_add_filter_context_to_zombie_list(_Inout_ net_ebpf_extension_wfp_filter_context_t* filter_context)
```
since this function can change the link entry in the filter context itself in line 1020